addons/crm/models/crm_lead.py

```diff
@@ -249,12 +268,38 @@ def _compute_team_id(self):
             if not lead.user_id:
                 continue
             user = lead.user_id
-            if lead.team_id and user in lead.team_id.member_ids | lead.team_id.user_id:
+            if lead.team_id and user in (lead.team_id.member_ids | lead.team_id.user_id):
```

Comment:
Looks like a fix that should be made in the stable version.
Or a useless change.